manifests/components/fluentd-es.jsonnet

```diff
@@ -80,7 +86,7 @@ local FLUENTD_ES_LOG_BUFFERS_PATH = "/var/log/fluentd-buffers";
                 runAsUser: 0,  // required to be able to read system-wide logs.
               },
               env_+: {
-                FLUENTD_OPT: "-q",
+                FLUENTD_OPT: "-o /tmp/fluentd.log --log-rotate-age 5 --log-rotate-size 104857600 --no-supervisor",
```

Comment:
i would suggest dropping the logs in `/opt/bitnami/fluentd/logs` which is a standard path for most bitnami images.

Reply:
Done.